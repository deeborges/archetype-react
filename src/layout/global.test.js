// teste de regrecao
import React from 'react';
import { render } from '@testing-library/react';

import LayoutGlobal from './global.layout';

test('renderizar o estilo global', () => {
	render(<LayoutGlobal />);
	expect(document.head).toMatchSnapshot();
});
