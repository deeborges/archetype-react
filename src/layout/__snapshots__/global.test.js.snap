// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderizar o estilo global 1`] = `
<head>
  <style
    data-styled="active"
    data-styled-version="5.2.1"
  >
    *{margin:0;padding:0;}
    *,*:before,*:after{box-sizing:inherit;}
    body,html{margin:0;font-size:62.5%;position:relative;}
    h1,h2,h3,h4,h5,h6{line-height:1.3;font-weight:700;-webkit-letter-spacing:2px;-moz-letter-spacing:2px;-ms-letter-spacing:2px;letter-spacing:2px;}
    h1 strong,h2 strong,h3 strong,h4 strong,h5 strong,h6 strong{color:#ffc107;}
  </style>
</head>
`;
